src/containers/permissions/assign-permissions-to-data-set/AssignPermissionsToDataSetModalBody.js

```diff
@@ -71,8 +72,9 @@ const AssignPermissionsToDataSetModalBody = ({
     .join(', ');
 
   const confirmText = assignPermissionsToAllProperties
-    ? `Please confirm you want to assign ${permissions} to "${targetDataSetTitle}" and all its columns.`
-    : `Please confirm you want to assign ${permissions} to "${targetDataSetTitle}".`;
+    ? `Please confirm you want to assign ${permissions} to "${targetDataSetTitles.join(', ')}" `
+      + 'datasets and all of their columns.'
```

Comment:
This could become pretty unreadable as data set names can be fairly large ...

> Please confirm you want to assign OWNER, READ, WRITE to "Shiproom: DemoAddresses", "Shiproom: DemoSentencedWith", "Shiproom: DemoArrestedBy", "Shiproom: DemoChargeEvents" datasets and all of their columns

I'm thinking we should turn this into a `<ul>`:

```
Please confirm you want to assign OWNER, READ, WRITE to the following data sets and all of their columns:
  - Shiproom: DemoAddresses
  - Shiproom: DemoSentencedWith
  - Shiproom: DemoArrestedBy
```